libs/agno/agno/tools/models_labs.py

```diff
@@ -1,124 +1,161 @@
 import json
 import time
 from os import getenv
-from typing import Optional
+from typing import Any, Dict, Optional
 from uuid import uuid4
 
 from agno.agent import Agent
-from agno.media import ImageArtifact, VideoArtifact
+from agno.media import AudioArtifact, ImageArtifact, VideoArtifact
 from agno.models.response import FileType
 from agno.tools import Toolkit
 from agno.utils.log import logger
 
 try:
     import requests
+    from requests.exceptions import RequestException
 except ImportError:
     raise ImportError("`requests` not installed. Please install using `pip install requests`")
 
+MODELS_LAB_URLS = {
+    "MP4": "https://modelslab.com/api/v6/video/text2video",
+    "MP3": "https://modelslab.com/api/v6/voice/music_gen",
+    "GIF": "https://modelslab.com/api/v6/video/text2video",
+}
+
+MODELS_LAB_FETCH_URLS = {
+    "MP4": "https://modelslab.com/api/v6/video/fetch",
+    "MP3": "https://modelslab.com/api/v6/voice/fetch",
+    "GIF": "https://modelslab.com/api/v6/video/fetch",
+}
+
 
 class ModelsLabTools(Toolkit):
     def __init__(
         self,
         api_key: Optional[str] = None,
-        url: str = "https://modelslab.com/api/v6/video/text2video",
```

Comment:
This is a breaking changes, so @mishramonalisha76 please make good notes about this in the changelog.

Reply:
yes I have added the details in changelog